File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Log_UpdateInformationService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Log_UpdateInformationService : BaseService<Log_UpdateInformation, ILog_UpdateInformationRepository>
    , ILog_UpdateInformationService
    {
    private readonly ILog_UpdateInformationRepository _Log_UpdateInformationRepository;
    public Log_UpdateInformationService(ILog_UpdateInformationRepository Log_UpdateInformationRepository) : base(Log_UpdateInformationRepository)
    {
    _Log_UpdateInformationRepository = Log_UpdateInformationRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Log_UpdateInformation.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Log_UpdateInformation } from './Log_UpdateInformation.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Log_UpdateInformationService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Update_ID', 'UpdateData_TableName', 'UpdateData_Id', 'UpdateDate', 'UpdateDateTime', 'UpdateDay', 'UpdateMonth', 'UpdateYear', 'UpdateSecond', 'UpdateMinute', 'UpdateHour', 'User_Id', 'LoginSession_Id'];

    List: Log_UpdateInformation[] | undefined;
    ListFilter: Log_UpdateInformation[] | undefined;
    FormData!: Log_UpdateInformation;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Log_UpdateInformation";
    }
}

